### DISCUSSION TITLE
Duplication without modification

### DISCUSSION THREAD

**Discussion by MrCasquette** [directus/directus#24686] [Feature Requests]
### Describe the Improvement

Is there a way to add a user experience enhancement, i.e. duplicate an element without modifying it first? For example, add a duplicate button next to the archive button. Or is it deliberate, to avoid specific errors?
Perhaps it would be useful to prohibit duplication of indexed fields?

**Comment by rijkvanzanten** [comment]
> Or is it deliberate, to avoid specific errors?

The main reason it doesn't exist now is for what you mentioned second: 

> Perhaps it would be useful to prohibit duplication of indexed fields?

If you were to duplicate in a collection that has a unique field in it, all duplications would fail 🤔 Would you imagine us hiding the duplicate button, or maybe render a modal with just the inputs that have to be changed before it can be saved? 

A similar question exists around manually entered primary keys; those can't be duplicated. The solution for unique fields and manually entered primary keys should be the same though, so maybe that's two birds in one stone 🐦🐦🪨
